### PR TITLE
bundle: add new recipe subscription

### PR DESCRIPTION
**Pull request by iamniting**
Signed-off-by: Nitin Goyal <nigoyal@redhat.com>

Bug: https://bugzilla.redhat.com/show_bug.cgi?id=2277766

### CODE REVIEW

File: hack/make-bundle-vars.mk [hunk 1]
```diff
@@ -126,9 +126,17 @@ PROMETHEUS_BUNDLE_IMG_TAG ?= main-d5fbc29
 PROMETHEUS_BUNDLE_IMG_LOCATION ?= quay.io/ocs-dev
 PROMETHEUS_BUNDLE_IMG ?= $(PROMETHEUS_BUNDLE_IMG_LOCATION)/$(PROMETHEUS_BUNDLE_IMG_NAME):$(PROMETHEUS_BUNDLE_IMG_TAG)
 
+RECIPE_BUNDLE_NAME ?= recipe
+RECIPE_BUNDLE_IMG_NAME ?= $(RECIPE_BUNDLE_NAME)-bundle
+RECIPE_BUNDLE_VERSION ?= v0.0.1
+RECIPE_BUNDLE_IMG_TAG ?= latest
+RECIPE_BUNDLE_IMG_LOCATION ?= quay.io/raghavendra_talur
```
Comment (Nikhil-Ladha) [inline hack/make-bundle-vars.mk:133]: Shouldn't we update this to ocs-dev?

Comment (Nikhil-Ladha) [reply hack/make-bundle-vars.mk:133]: Anyway, the CI seems to be working so, fine 😅

**Review by Nikhil-Ladha — APPROVED** [review]
/lgtm

File: hack/make-bundle-vars.mk (hunk 1 above)
Comment (iamniting) [reply hack/make-bundle-vars.mk:133]: currently we dont have a image pushed, We don't want to delay the PR further. I will update once we have the image.

**Pull request merged**